Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -80,9 +92,15 @@ @implementation SessionDelegate {
     std::vector<xr::System::Session::Frame::View>* activeFrameViews;
 
     NSLock* planeLock;
+    NSLock* meshLock;
     std::set<ARPlaneAnchor*,ARAnchorComparer> updatedPlanes;
     std::vector<ARPlaneAnchor*> deletedPlanes;
     bool planeDetectionEnabled;
+    API_AVAILABLE(ios(13.4))
+    std::set<ARMeshAnchor*,ARAnchorComparer> updatedMeshes;
+    API_AVAILABLE(ios(13.4))
+    std::vector<ARMeshAnchor*> deletedMeshes;
```

Comment:
I think in this case, since this is just part of the implementation details, you can just do:
```suggestion
#if (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_4)
    std::set<ARMeshAnchor*,ARAnchorComparer> updatedMeshes;
    std::vector<ARMeshAnchor*> deletedMeshes;
#endif
```